pyproject.toml

```diff
@@ -110,7 +110,7 @@ databricks-cli = { version = "^0.17.3", optional = true }
 databricks-sql-connector = { version = "2.8.0", optional = true }  # Awaiting snowflake-connector-python to support urllib3>=2.0
 fastapi = { version =  "^0.96.0", optional = true }
 featurebyte-freeware = { version = "^0.2.16", optional = true }
-gevent = {version = "^22.10.2", optional = true}
+gevent = {version = "*", optional = true}
```

Comment:
Curious does this allow any version? Is it possible to specify a stricter specification that is closer to what we have tested?

Reply:
sure doing the changes, but I think the upgrade caused the test to all fail